tests/read_pr.py

```diff
@@ -0,0 +1,175 @@
+import glob
+
+
+class ReadPR:
+    """
+    Recursive find '*** Keywords ***' text relationship for the files that were changed.
+    For each match retrieve and report what are the tags that we should run the test suite.
+    """
+
+    def __init__(self):
+        self.keyword_key = '*** Keywords ***'
+        self.magic_tag_separator = '_______'
+        self.magic_tag = '[Tags]'
+
+    def run(self):
+        data = self.get_diff()
+        files_changed = []
+        for line in data:
+            if line.startswith('diff --git '):
+                files_changed.append(line)
+        infos = []
+        for f in files_changed:
+            is_resource = f and f.endswith('.resource')
+            is_robot = f.endswith('.robot')
+            if is_robot or is_resource:
```

Comment:
do we need line 22 23 and 24. line can be be sorted in below code :

`    for f in files_changed:
        if f.endswith(('.robot', '.resource')):
            infos.append(self.get_sections_info(f))`

Reply:
Done